Resources/Textures/Objects/Misc/bureaucracy.rsi/meta.json

```
@@ -297,7 +297,7 @@
       "name": "stamp-trader"
     },
     {
-      "name": "stamp-warden"
+        "name": "stamp-warden"
```

```suggestion
      "name": "stamp-warden"
```
You seem to have accidently added a space here.